Office 365/o365/_meta/fields.yml

```
@@ -218,6 +218,15 @@ office365.audit.object_id:
   description: "For Exchange admin audit logging, the name of the object that was modified by the cmdlet. For SharePoint activity, the full URL path name of the file or folder accessed by a user. For Azure AD activity, the name of the user account that was modified."
   type: keyword
 
+office365.virus_info:
+  name: office365.virus_info
+  description: "VirusInfo"
+  type: keyword
+  
+ office365.virus_vendor:
```

Please remove this trailing space
```suggestion
office365.virus_vendor:
```